functions/AutoRemediateOrchestrator.js

```diff
@@ -15,40 +14,35 @@ module.exports.handler = (event, context, callback) => {
       'The %s is not supported. Exiting gracefully ...',
       AutoRemediate
     )
-    callback(null)
-    return
+    return null
```

Comment:
♻️ While it probably doesn't matter, this is not actually logically equivalent as previously it returned `undefined` to the `callback` as the second argument is what is returned and as it wasn't passed then it would return `undefined`. The suggestion below is logically equivalent. 

```suggestion
    return;
```